### PR TITLE
LPS-170469 [PRM] Adding more fields into Activity Form and Refactoring some functionality

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/utils/getEntriesByListTypeDefinitions.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 
 import ListTypeDefinition from '../../../common/interfaces/listTypeDefinition';
-interface EntryField {
+export interface EntryField {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/utils/getEntriesByListTypeDefinitions.ts:13]:
```suggestion
interface EntryField {
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by biellopes00** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
ci:forward

**Pull request closed without merging**